index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
         <section class="social-details">
           <h5>LET'S CONNECT</h5>
           <div class="social-wrapper">
-            <span class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></span>
-            <span class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></span>
-            <span class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></span>
+            <ul>
+              <li class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></li>
+              <li class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></li>
+              <li class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></li>
+            </ul>
           </div>
         </section>
+        <!-- Projects Section -->
+        <section class="work">
+          <div class="card">
+            <div class="card-container">
+              <div class="card-1"></div>
+              <div class="card-desc">
+                <h2>Tonic</h2>
+                <ul>
+                  <li>Canopy</li>
+                  <li>Back End Dev</li>
+                  <li>2022</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="card">
+            <div class="card-2"></div>
+              <div class="card-desc">
+                <h2>Muilti-Post Stories</h2>
+                <ul>
+                  <li>FACEBOOK</li>
+                  <li>Full stack Dev</li>
+                  <li>2023</li>
+                </ul>
+                <div class="desc">
+                  <p>A daily selection of privately personalized reads, no accounts or sign-ups required</p>
+                  <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                  </ul>
+                  <button type="button">See Project</button>
```

Comment:
- [ ] The button (e.g., "See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover, and Pressed (see design). Please add the Active, Hover & Pressed design as follows at all places where the button is present as follows:-
![nynrtb](https://user-images.githubusercontent.com/26767383/153623252-75501d9e-fc64-41d1-97e4-f16521d0015e.PNG)

Reply:
I have added the hover effect on the buttons